packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -57,6 +57,7 @@ class Braze {
       baseUrl: this.endPoint,
       allowUserSuppliedJavascript: this.allowUserSuppliedJavascript,
     });
+    window.braze.addSdkMetadata([window.braze.BrazeSdkMetadata.CDN, 'rud']);
```

Comment:
Can you also share the Braze documentation reference for this?

Reply:
This is internal metadata, we don't have docs for this,. unfortunately.